routes/render.go

```diff
@@ -19,7 +19,14 @@ import (
 	"github.com/tdewolff/minify/v2/minify"
 )
 
-func colour(i int) string {
+func colour(rank int, points int, display string) string {
+	if display == "points" {
+		return colourPoints(points)
+	}
+	return colourRank(rank)
+}
+
+func colourRank(i int) string {
```

Comment:
These really ought to be local to where they're used and not global template functions, but that's not your fault, it was already like that. Would the logic look too messy directly in the template?

Reply:
Yes, it would be too messy directly in the template. Template tags are not the place for complex logic. I've split it into its own file. Still global, but a separate go file.